src/controllers/users.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+const usersService = require('../services/users');
+
+const getAll = async(req, res) => {
+  const users = await usersService.getAllUsers();
+
+  res.send(users.map(user => usersService.normalize(user)));
+};
+
+const addUser = async(req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = await usersService.createUser(name);
+
+  res.status(201);
+  res.send(newUser);
+};
+
+const getOneUser = async(req, res) => {
+  const { userId } = req.params;
```

Comment:
Let's check required data before using business logic

```suggestion
  const { userId } = req.params;
  if (!userId) {
    some error
  }
```